src/App.js

```diff
@@ -8,6 +8,7 @@ function App() {
   return (
     <Routes>
       <Route exact path="/" element={<Rockets />} />
+      <Route path="/Rockets" element={<Rockets />} />
```

Comment:
Here, you don't need to repeat the line twice. Please remove it to apply DRY rule.